python/sglang/srt/managers/tp_worker.py

```diff
@@ -197,7 +197,7 @@ def update_weights_from_distributed(
 
     def update_weights_from_tensor(self, recv_req: UpdateWeightsFromTensorReqInput):
         success, message = self.model_runner.update_weights_from_tensor(
-            recv_req.name, recv_req.tensor
+            MultiprocessingSerializer.deserialize(recv_req.serialized_named_tensors)
```

Comment:
WIll this work for inter-node cases? For example, if tp_size is greater than 8

Reply:
Yes